templates/repo/issue/view_content/comments.tmpl

```diff
@@ -582,5 +583,25 @@
 				{{end}}
 			</span>
 		</div>
+	{{else if eq .Type 29}}
+		<div class="timeline-item event" id="{{.HashTag}}">
+			<span class="badge">{{svg "octicon-repo-force-push" 16}}</span>
+			<a class="ui avatar image" href="{{.Poster.HomeLink}}">
+				<img src="{{.Poster.RelAvatarLink}}">
+			</a>
+			<span class="text grey">
+				<a href="{{.Poster.HomeLink}}">{{.Poster.GetDisplayName}}</a>
+				{{ if .IsForcePush }}
+					{{ $oldCommitLink:= printf "%s/%s/%s/commit/%s" AppSubUrl $.Issue.PullRequest.BaseRepo.OwnerName $.Issue.PullRequest.BaseRepo.Name .OldCommit }}
```

Comment:
This should probaly be calculated on the router and exposed as a property. This is business logic and IMHO doesn't belong in the template.